{
	"title": "CVE-2023-43789",
	"cve": "CVE-2023-43789",
	"notes": {
		"description": "A vulnerability was found in libXpm where a vulnerability exists due to a boundary condition, a local user can trigger an out-of-bounds read error and read contents of memory on the system.",
		"published": "2023-10-12",
		"modified": "2023-12-05"
	},
	"references": [
		{
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/I3KFUQT42R7TB4D7RISNSBQFJGLTQGUL/",
			"description": "FEDORA:FEDORA-2023-25329c196b"
		},
		{
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2242249",
			"description": "MISC:RHBZ#2242249"
		},
		{
			"url": "https://access.redhat.com/security/cve/CVE-2023-43789",
			"description": "MISC:https://access.redhat.com/security/cve/CVE-2023-43789"
		},
		{
			"url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/63IBRFLQVZSMOAZBZOBKFWJP26ILRAGQ/",
			"description": "MISC:https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/63IBRFLQVZSMOAZBZOBKFWJP26ILRAGQ/"
		}
	]
}
